src/api/procedures/modifyInstructionAffirmation.ts

```diff
@@ -144,7 +144,9 @@ const assertReceipts = async (
 
     const { address: signerAddress } = asAccount(signer, context);
 
-    const isAllowedSigner = allowedSigners.some(({ address }) => signerAddress === address);
+    const isAllowedSigner = allowedSigners
+      ? allowedSigners.some(({ address }) => signerAddress === address)
+      : true;
```

Comment:
```suggestion
      : false;
```
Shouldn't this be false? Not sure again, better to confirm with core team

Reply:
If it was false then it would be impossible to affirm any instruction without a venue, which can't be right.